openff/interchange/interop/internal/amber.py

```diff
@@ -0,0 +1,630 @@
+        [excluded_atoms_list.append(_) for _ in tmp]
+
+    return number_excluded_atoms, excluded_atoms_list
+
+
+def to_prmtop(interchange: "Interchange", file_path: Union[Path, str]):
+    """
+    Write a .prmtop file. See http://ambermd.org/prmtop.pdf for details.
+
+    """
+    if isinstance(file_path, str):
+        path = Path(file_path)
+    if isinstance(file_path, Path):
+        path = file_path
+
+    if interchange["vdW"].mixing_rule != "lorentz-berthelot":
+        raise Exception
+
+    with open(path, "w") as prmtop:
+        import datetime
+
+        now = datetime.datetime.now()
+        prmtop.write(
+            "%VERSION  VERSION_STAMP = V0001.000  DATE = "
+            f"{now.month:02d}/{now.day:02d}/{(now.year % 100):02}  "
+            f"{now.hour:02d}:{now.minute:02d}:{now.second:02d}\n"
+            "%FLAG TITLE\n"
+            "%FORMAT(20a4)\n"
+            "\n"
+        )
+
+        from openff.interchange.interop.internal.gromacs import _build_typemap
+
+        typemap = _build_typemap(interchange)  # noqa
+
+        potential_key_to_atom_type_mapping: Dict[PotentialKey, int] = {
+            key: i for i, key in enumerate(interchange["vdW"].potentials)
+        }
+        atom_type_indices = [
+            potential_key_to_atom_type_mapping[potential_key]
+            for potential_key in interchange["vdW"].slot_map.values()
+        ]
+
+        potential_key_to_bond_type_mapping: Dict[PotentialKey, int] = {
+            key: i for i, key in enumerate(interchange["Bonds"].potentials)
+        }
+
+        potential_key_to_angle_type_mapping: Dict[PotentialKey, int] = {
+            key: i for i, key in enumerate(interchange["Angles"].potentials)
+        }
+
+        potential_key_to_dihedral_type_mapping: Dict[PotentialKey, int] = {
+            key: i for i, key in enumerate(interchange["ProperTorsions"].potentials)
+        }
+
+        # Track bonds and angles here also to ensure the 1-2 and 1-3 exclusions are
+        # properly applied.
+        known_14_pairs = list()
+
+        bonds_inc_hydrogen = list()
+        bonds_without_hydrogen = list()
+
+        for bond, key in interchange["Bonds"].slot_map.items():
+            bond_type_index = potential_key_to_bond_type_mapping[key]
+
+            atom1 = interchange.topology.mdtop.atom(bond.atom_indices[0])
+            atom2 = interchange.topology.mdtop.atom(bond.atom_indices[1])
+
+            bond_indices = sorted(bond.atom_indices)
+
+            if atom1.element.atomic_number == 1 or atom2.element.atomic_number == 1:
+                bonds_inc_hydrogen.append(bond_indices[0] * 3)
+                bonds_inc_hydrogen.append(bond_indices[1] * 3)
+                bonds_inc_hydrogen.append(bond_type_index + 1)
+            else:
+                bonds_without_hydrogen.append(bond_indices[0] * 3)
+                bonds_without_hydrogen.append(bond_indices[1] * 3)
+                bonds_without_hydrogen.append(bond_type_index + 1)
```

Comment:
```suggestion
            bonds_list = bonds_inc_hydrogen if (atom1.element.atomic_number == 1 or atom2.element.atomic_number == 1) else bonds_without_hydrogen:
            
            bonds_list.append(bond_indices[0] * 3)
            bonds_list.append(bond_indices[1] * 3)
            bonds_list.append(bond_type_index + 1)
```

might help avoid a copy and paste!